### PR TITLE
working ampliconclip tool

### PR DESCRIPTION
**Pull request by bwlang**
(10x faster than iVar trim, able to trim both sides to remove primer-dimer aligments)

FOR CONTRIBUTOR:
* [X] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [X] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [X] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

### CODE REVIEW

File: tool_collections/samtools/samtools_ampliconclip/samtools_ampliconclip.xml [hunk 1]
```diff
@@ -0,0 +1,83 @@
+<tool id="samtools_ampliconclip" name="Samtools ampliconclip" version="1" profile="@PROFILE@">
```
Comment (bgruening) [inline tool_collections/samtools/samtools_ampliconclip/samtools_ampliconclip.xml:1]: why version 1 and not the samtools version?

File: tool_collections/samtools/samtools_ampliconclip/samtools_ampliconclip.xml [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+        <param name="strand" argument="--strand" type="boolean" checked="false" truevalue="--strand" falsevalue="" label="only clip reads that match bed file strand annotation" />
+        <param name="both_ends" argument="--both-ends" type="boolean" checked="false" truevalue="--both-ends" falsevalue="" label="clip both ends of reads (false = 5' only)" />
+        <param name="no_excluded" argument="--no-excluded" type="boolean" checked="false" truevalue="--no-excluded" falsevalue="" label="don't write excluded reads to output (default = write all)" />
+        <param name="min_length" argument="--fail-len" type="integer" min="0" optional="true" label="Min Read length" help="mark reads QCFAIL at this length or shorter after clipping" />
+        <param name="tolerance" argument="--tolerance" type="integer" value="5" min="0"  label="Tolerance" help="match region within this number of bases, default 5." />
+    
+    </inputs>
+    <outputs>
+        <data name="output_bam" format="bam" label="${tool.name} on ${on_string}" />
```
Comment (bgruening) [inline tool_collections/samtools/samtools_ampliconclip/samtools_ampliconclip.xml:40]: The label here is the default I think, so you can remove it.

**Review by bgruening — APPROVED** [review]

File: tool_collections/samtools/samtools_ampliconclip/samtools_ampliconclip.xml [hunk 3]
```diff
@@ -0,0 +1,83 @@
+<tool id="samtools_ampliconclip" name="Samtools ampliconclip" version="1.13" profile="@PROFILE@">
```
Comment (bernt-matthias) [inline tool_collections/samtools/samtools_ampliconclip/samtools_ampliconclip.xml:1]: Please use the version token for the tool version.

Comment (bwlang) [reply tool_collections/samtools/samtools_ampliconclip/samtools_ampliconclip.xml]: doh - forgot about that token!
thanks to @bgruening for adding it.

Comment (bgruening) [reply tool_collections/samtools/samtools_ampliconclip/samtools_ampliconclip.xml]: I take this as you are ok with it and will merge :)

**Review by bgruening — APPROVED** [review]

**Comment by bgruening** [comment]
fyi: @wm75

**Pull request merged**